### ISSUE TITLE
Session ID

### ISSUE THREAD

**Issue by grumpykiwi** [Blazored/SessionStorage#65]
**Please be sure to check the readme file before raising an issue.**
Does your component expose the Session ID?

I am converting an old .Net Framework app to Blazor Server .Net 6. There is some code that has to keep track of the Session ID as a lookup value to a SQL table. I would prefer to not have to rewrite this code as it is a bit of a kludge anyway. but the functionality was a must have for the client.

Essentially allows an admin to "impersonate" a regular user to assist with questions about their benefits etc. The admin gets to see what the regular user sees. Session ID is used to keep track of whether a use is being impersonated or not, and helps to restore everything when impersonation ends. That is the abridged version. It is way more complex, business rule wise.

Anyway, any help would be appreciated.

Thanks

Mark

**Comment by chrissainty** [comment]
What do you mean by session ID?

**Comment by grumpykiwi** [comment]
In the old .Net framework 4.8 code I am using the following to track a user's session in a SQL table with some other values

var sessionId = HttpContext.Current.Session.SessionID;

Code would then fetch a record if if existed from the table, or create the record if it didn't. 

In the rewrite I am working on, I was looking for something similar as a quick and dirty replacement. I had to go and look at this code up again and it looks like I found a different way of handling this.

I am still looking for a drop in replacement for Elmah though :-)

Thanks for replying.

Cheers


Mark
